client/components/forms/form-radios-bar/index.jsx

```diff
@@ -0,0 +1,45 @@
+/** @format */
+/**
+ * External dependencies
+ */
+import React from 'react';
+import PropTypes from 'prop-types';
+import classnames from 'classnames';
+
+/**
+ * Internal dependencies
+ */
+
+import FormLabel from 'components/forms/form-label';
+import FormRadio from 'components/forms/form-radio';
+import FormRadioWithThumbnail from 'components/forms/form-radio-with-thumbnail';
+
+const FormRadiosBar = ( { isThumbnail, checked, onChange, items } ) => {
+	const addProps = ( Component, item, key ) =>
+		<Component key={ key } checked={ checked === item.value } onChange={ onChange } { ...item } />;
```

Comment:
Wanted to avoid the duplication but given that it's such a small block I think it's a fair point to just duplicate it to reduce complexity.